build.sbt

```diff
@@ -95,15 +89,15 @@ lazy val noPublish = List(
   skip in publish := true
 )
 
-// not publishing the root project
-noPublish
+lazy val root = project
+  .in(file("."))
+  .settings(
+    noPublish
+  )
```

Comment:
can you preserve `name := "metalsRoot"` here?

The reason why it was there is that sbt uses a bogus name for the root project, because the default name (the directory) is already taken by a project.

Reply:
I don't know why exactly, but when I started working on this PR, `name := "metalsRoot"` didn't work and that bodus name appeared, so I added this explicit root project declaration. Now it's called `root`, which I think is quite standard naming in multi-build projects. But if you prefer `metalsRoot`, I can rename it, no problem.

Reply:
`metalsRoot` is better since this name appears in IntelliJ "Open recent"

![screen shot 2018-04-03 at 15 43 13](https://user-images.githubusercontent.com/1408093/38252970-c72da9fe-3755-11e8-9930-586f3c1191da.png)

Reply:
ah, I see.
No problem, as long as the horrible generated name doesn't show up, I'm fine with "root"!

Reply:
Nevermind, what @olafurpg said :D

Reply:
OK 👌